mq/publishers.go

```diff
@@ -13,6 +14,8 @@ import (
 	"golang.org/x/exp/slog"
 )
 
+const batchSize = 20
```

Comment:
should be read from config, we should also have a way to override the batch updates

Reply:
It's good to set a switch to turn on/off batch peerUpdate. 
Regarding the batchsize,  it's necessary to open it to end user?  Please help confirm.

Reply:
yes, allowing it to be configured will be flexible for us

Reply:
Added the peerUpdate batch size to config. Please help review.